jccli/cli/sync.py

```diff
@@ -260,7 +88,7 @@ def sync_groups(ctx, groups):
             new_group = {}
             new_group['name'] = group_name
             new_group['type'] = group_type
-            LOGGER.info(f"create {' '.join(group_type.split('_'))}: {group_name}")
+            click.echo(f"create {' '.join(group_type.split('_'))}: {group_name}")
```

Comment:
is there a reason why you change logger to click.echo?  I prefer logger so we an enable logging levels